platforms/android/library/build.gradle

```diff
@@ -54,6 +54,8 @@ android {
     viewBinding {
         enabled = true
     }
+
+    ndkVersion = "22.1.7171670"
```

Comment:
@jmartinesp this part was particularly unsure for me. I needed it for the build to work, but is it right?

Reply:
In my case gradle seems to just ignore it, I don't have that version installed and it's building just fine. I guess it's using what's on `bindings/wysiwyg-ffi/.cargo/config.toml` instead. It's weird that you have to add it to be able to build the lib.